receiver/dockerstatsreceiver/testdata/mock/two_containers/expected_metrics.yaml

```diff
@@ -562,7 +562,7 @@ resourceMetrics:
             sum:
               aggregationTemporality: 2
               dataPoints:
-                - asInt: "352256"
+                - asInt: "425984"
```

Comment:
The description of this metric is now a bit misleading, can you change that in `metadata.yml`?

It can probably just be "Total memory usage of the container".

Reply:
Perhaps replacing the `total cache` ?:
`Memory usage of the container. This excludes the inactive file memory.`

Or just mentioning that excludes "cache" like Docker CLI [docs](https://docs.docker.com/engine/reference/commandline/stats/#description) does in the note?
`Memory usage of the container. This excludes the cache.`

WDYT

Reply:
Either one works for me

Reply:
done in [a756251](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21166/commits/a75625150da0a1b69236296dc7356a66b9ceb770)